src/Datadog.Trace/RuntimeMetrics/PerformanceCountersListener.cs

```diff
@@ -130,7 +130,7 @@ protected virtual void InitializePerformanceCounters()
                 // Catching error UnauthorizedAccessException: Access to the registry key 'Global' is denied.
                 // The 'Global' part seems consistent across localizations
 
-                Log.Error(ex, "The process does not have sufficient permissions to read performance counters. Please refer to https://docs.datadoghq.com/tracing/runtime_metrics/dotnet/#additional-permissions-for-iis to learn how to grant those permissions.");
+                Log.Error(ex, "The process does not have sufficient permissions to read performance counters. Please refer to https://dtdg.co/net-runtime-metrics to learn how to grant those permissions.");
```

Comment:
Can we use `dotnet` instead of `net`?

Reply:
Refreshed the browser and noticed this is already merged 🤦🏽‍♂️

Reply:
That's okay, we can always create a new link 😆